### PR TITLE
Feature/get input data

### PR DESCRIPTION
**Pull request by elisehuard**
Get input data per category
In handler:
GET "/forecasts/:id/:version/input/:category"

### CODE REVIEW

File: src/witan/app/handler.clj [hunk 1]
```diff
@@ -161,7 +161,12 @@
...
                                 (forecast/input-data {:id id :version version :category category :user-id (:identity request)}))
-                   (sweet/POST* "/tag" []
+                   (sweet/GET* "/forecasts/:id/:version/input/:category" []
+                                :summary "get input data item"
+                                :path-params [id :- java.util.UUID
+                                              version :- java.lang.Long
+                                              category :- String]
+                                (forecast/input-data {:id id :version version :category category}))                               (sweet/POST* "/tag" []
                                 :summary "Creates a new tag from a forecast id and version"
                                 (not-implemented))
```
Comment (acron0) [inline src/witan/app/handler.clj]: I think ln170 - ln171 want to be removed?

Comment (elisehuard) [inline src/witan/app/handler.clj]: don't know what happened there, it's a case of an end-of-line disappearing (tag url at the end of the line)

Comment (elisehuard) [inline src/witan/app/handler.clj]: re-added end of line

**Comment by acron0** [comment]
lgmt :shipit:

**Pull request merged**